src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -486,13 +486,18 @@ void EIRPlantLoopHeatPump::calcAvailableCapacity(EnergyPlusData &state, Real64 c
             }
         }
         if (this->heatRecoveryHeatPump) {
+            this->calcLoadSideHeatTransfer(state, availableCapacity);
+            this->calcPowerUsage(state);
+            Real64 sourceSideHeatTransfer = this->calcQsource(availableCapacity * partLoadRatio, this->powerUsage);
```

Comment:
Adds the steps required to calculate the power usage so that the source side heat transfer is estimated correctly.